<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-20T11:59:10 GMT-07:00" name="Default suite" finished-at="2025-07-20T11:59:26 GMT-07:00" duration-ms="15978">
    <groups>
    </groups>
    <test started-at="2025-07-20T11:59:10 GMT-07:00" name="Default test" finished-at="2025-07-20T11:59:26 GMT-07:00" duration-ms="15978">
      <class name="api.test.DDTCombined">
        <test-method signature="testCreateAndDeleteUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:api.test.DDTCombined@2ceb80a1]" started-at="2025-07-20T11:59:11 GMT-07:00" name="testCreateAndDeleteUser" data-provider="Data" finished-at="2025-07-20T11:59:17 GMT-07:00" duration-ms="6075" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[1010]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ravi1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Avi1]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Goel1]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[a@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Ravi@1]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1123456789]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1431)
at org.testng.Assert.assertEquals(Assert.java:1395)
at org.testng.Assert.assertEquals(Assert.java:1441)
at api.test.DDTCombined.testCreateAndDeleteUser(DDTCombined.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAndDeleteUser -->
        <test-method signature="testCreateAndDeleteUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:api.test.DDTCombined@2ceb80a1]" started-at="2025-07-20T11:59:17 GMT-07:00" name="testCreateAndDeleteUser" data-provider="Data" finished-at="2025-07-20T11:59:22 GMT-07:00" duration-ms="4654" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1020]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ravi2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Avi2]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Goel2]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[b@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Ravi@2]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1223456789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAndDeleteUser -->
        <test-method signature="testCreateAndDeleteUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:api.test.DDTCombined@2ceb80a1]" started-at="2025-07-20T11:59:22 GMT-07:00" name="testCreateAndDeleteUser" data-provider="Data" finished-at="2025-07-20T11:59:26 GMT-07:00" duration-ms="4409" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1030]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ravi3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Avi3]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Goel3]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[c@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Ravi@3]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1233456789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAndDeleteUser -->
      </class> <!-- api.test.DDTCombined -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
